cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -463,22 +472,24 @@ class alignas(16) column_device_view : public detail::column_device_view_base {
    */
   struct index_element_fn {
     template <typename IndexType,
-              CUDF_ENABLE_IF(is_index_type<IndexType>() and std::is_signed_v<IndexType>)>
+              CUDF_ENABLE_IF(is_index_type<IndexType>() and cuda::std::is_signed_v<IndexType>)>
     __device__ size_type operator()(column_device_view const& indices, size_type index)
     {
       return static_cast<size_type>(indices.element<IndexType>(index));
     }
 
     template <typename IndexType,
               typename... Args,
-              CUDF_ENABLE_IF(not(is_index_type<IndexType>() and std::is_signed_v<IndexType>))>
+              CUDF_ENABLE_IF(not(is_index_type<IndexType>() and cuda::std::is_signed_v<IndexType>))>
     __device__ size_type operator()(Args&&... args)
     {
       CUDF_UNREACHABLE("dictionary indices must be a signed integral type");
     }
   };
 
  public:
+#ifndef CUDF_RUNTIME_JIT
```

Comment:
Does this break syntax highlighting in VSCode? I have seen before that blocks like this might appear in gray because the editor thinks it's "false" and thus inactive.

Reply:
It is not defined by default, only when compiling with NVRTC, so that block will always be true in VSCode and syntax highlighting will always work.